migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc

```diff
@@ -0,0 +1,94 @@
+[id="migrating-openshift-4_2-to-4"]
+= Migrating {product-title} 4.2 to 4.2
+include::modules/common-attributes.adoc[]
+:context: ocp-4_2-4_2
+:ocp-4_2-4_2:
+
+toc::[]
+
+You can migrate application workloads from {product-title} 4.2 to 4.2 with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
+
+The CAM tool's web console and API, based on Kubernetes custom resources, enable you to migrate stateful and stateless application workloads at the granularity of a namespace.
+
+You can migrate data to a different storage class, for example, from Red Hat Gluster Storage or NFS storage to Red Hat Ceph Storage.
+
+.Prerequisites
+
+* The source cluster must be {product-title} 4.2.
+* The target cluster must be {product-title} 4.2.
+* You must have xref:../../cli_reference/openshift_cli/administrator-cli-commands.html#policy[`cluster-admin` privileges] on all clusters.
+* You must have a replication repository that supports the S3 API and is accessible to the source and target clusters.
+* If your application uses images from the `openshift` namespace, the required versions of the images must be present on the target cluster. If not, you must xref:../../openshift_images/image-streams-manage.html#images-imagestreams-update-tag_image-streams-managing[update the `imagestreamtags` references] to use an available version that is compatible with your application.
```

Comment:
As the source and target clusters are the same version 4.2, this shouldn't happen

Reply:
I will remove this section.